andela/travel_tool_front
# #160685068 Get user friendly trip type message

#### What does this PR do?

- This PR ensure that a user friendly message is diplayed in the trip type column. 

#### Description of Task to be completed?
- Given that I am a user, the trip type should be user friendly. Leaving trip type as multi or oneWay is not friendly. 

#### How should this be manually tested?
- git clone https://github.com/andela/travel_tool_front.git
- git checkout ch-fix-trip-type-160685068
- yarn install
- Run the server under develop branch
- yarn start
- Login in the system 
- Create several requests of multi, return and oneway
- Check the trip column section on messages received

#### Any background context you want to provide?

- Information received from backend as trip type is in form of variables for example oneWay. To make it user friendly, we change it to Oneway. 

#### What are the relevant pivotal tracker stories?
[#160685068](https://www.pivotaltracker.com/story/show/160685068)

#### Screenshots (if appropriate)
<img width="1374" alt="screen shot 2018-09-24 at 20 06 38" src="https://user-images.githubusercontent.com/32410518/45967022-68de5100-c035-11e8-8046-5b773fc22692.png">



#### Questions:
